### ISSUE TITLE
Foreign key model disconnects if there are no changes in the form

### ISSUE THREAD

**Issue by kevinob11** [daniyalzade/django_reverse_admin#33]
I'm on django version 2.1.12 and python version 3.7.3. I have an address nested on another model using reverse-django-admin. When I edit the main model if I don't make any changes to the address then the relation is lost (foreign key column is set to null) if I do make changes to the address then the relation is maintained. Below is my config:

```
class Address(BaseModel):
    addr1 = models.CharField(max_length=250)
    addr2 = models.CharField(max_length=250, blank=True, default='')
    city = models.CharField(max_length=250)
    state = models.CharField(choices=US_STATES, max_length=250)
    county = models.CharField(max_length=250, blank=True, default='')
    zip_code = USZipCodeField()

    class Meta:
        verbose_name_plural = 'Addresses'

    def __str__(self):
        return self.addr1

class Agency(BaseModel):
    name = models.CharField(max_length=250)
    address = models.ForeignKey(Address, on_delete=models.PROTECT, related_name='address')
```

Any help would be appreciated!

**Comment by kevinob11** [comment]
If I comment out these lines it works:

https://github.com/daniyalzade/django_reverse_admin/blob/master/django_reverse_admin/__init__.py#L50-L53

I'm assuming that breaks something else, just as a heads up.

**Comment by BoringOwlCompany** [comment]
Any update on this ?

**Comment by daniyalzade** [comment]
This was fixed with version 2.7.0 with the code changes on - https://github.com/daniyalzade/django_reverse_admin/pull/44 

Please let me know if you are still seeing the issue @kevinob11 / @BoringOwlCompany
